app/scripts/transportapi.js

```diff
@@ -142,6 +142,20 @@
       return transportAPIcall(url, successFunction);
     },
 
+    busPerformance: function(codes, successFunction, timeout) {
+      var codesParam = codes.join(',');
+
+      var url = transportAPIbaseURL +
```

Comment:
You are right, I'll make a re-factoring PR fixing this soon.
The problem is that all the functions have to be changed and the base one too, so it is a re-factoring.